operations/app/terraform/vars/demo/locals.tf

```diff
@@ -3,16 +3,17 @@ locals {
   is_temp_env = true
   address_id  = var.address_id
   init = {
-    environment         = var.environment
-    location            = "eastus"
-    is_metabase_env     = false
-    random_id           = local.random_id
-    resource_group_name = "prime-data-hub-${var.environment}"
-    resource_prefix     = "pdh${var.environment}"
-    okta_redirect_url   = "https://${var.environment}.prime.cdc.gov/download"
-    okta_base_url       = "hhs-prime.oktapreview.com"
-    fn_OKTA_clientId    = "xxx"
-    OKTA_scope          = "simple_report_dev"
+    environment           = var.environment
+    location              = "eastus"
+    is_metabase_env       = false
+    random_id             = local.random_id
+    resource_group_name   = "prime-data-hub-${var.environment}"
+    resource_prefix       = "pdh${var.environment}"
+    okta_redirect_url     = "https://${var.environment}.prime.cdc.gov/download"
```

Comment:
I wonder how this worked before since prod.prime.cdc.gov/download is not a real url

Reply:
Oops, please dismiss, did not mean to make this into a review, was supposed to be a comment